scripts/create_issues_new_exercise.sh

```diff
@@ -0,0 +1,153 @@
+    echo "TOKEN=\"your_token\""
+    echo "OWNER=\"repo_owner\""
+    echo "REPO=\"repo_name\""
+fi
+
+. "$HOME/.exercism-version-update-issue-script-settings.sh"
+
+if [[ -z "$TOKEN" ]]; then
+    echo "Please insert your personal token into \".exercism-version-update-issue-script-settings.sh\"."
+    exit 1
+elif [[ -z "$OWNER" ]]; then
+    echo "Please insert the repo owner into \".exercism-version-update-issue-script-settings.sh\"."
+    exit 1
+elif [[ -z "$REPO" ]]; then
+    echo "Please insert the name of the repo into \".exercism-version-update-issue-script-settings.sh\"."
+    exit 1
+fi
+
+
+# Fetch issues opened by us to avoid duplicate issues
+open_issues=$(curl --silent -HAuthorization:token\ ${TOKEN} https://api.github.com/repos/${OWNER}/${REPO}/issues\?state=open\&labels=implement+exercise | jq -r '.[] |(.title | split(":")[0]) + " Issue Title: " + .title + " (" + (.comments|tostring) +" comments)" + ", URL: " + .html_url')
+
+count=0
+for exercise in $problem_spec_exercises; do
+  if echo "$track_exercise_slugs" | grep -q "$exercise" ; then
+    continue
+  elif echo "$track_deprecated_exercises" | grep -q "$exercise"; then
+    continue
+  elif echo "$track_foregone_exercises" | grep -q "$exercise"; then
+    continue
+  fi
+  (( count+=1 ))
+    if echo "$open_issues" | grep --quiet "^$exercise "; then
+        echo ">>>>> $exercise: implement exercise"
+        echo "The following issue(s) are currently open for exercise $exercise in ${REPO} with label <implement exercise>:"
+        echo "$open_issues" | grep "^$exercise " | cut -d' ' -f2-
+        echo "Please check manually if a new issue may be opened."
+        read -p "Press enter to continue."
+        echo
+        continue
+    else
+        printf ">>>>> Exercise $exercise is not yet implemented in the Java track.\n"
+        title="$exercise: implement exercise"
+        body="The exercise **$exercise** has not been implemented yet for the Java track. \nThe description of the exercise can be found in the [problem specification repository](https://github.com/exercism/problem-specifications/tree/master/exercises/$exercise). \n\nHow to implement a new exercise for the Java track is described in detail in [CONTRIBUTING.md](https://github.com/exercism/java/blob/master/CONTRIBUTING.md#adding-a-new-exercise). Please have a look there first before starting working on the exercise. \nAlso please make sure it is clear that you are currently working on this issue, either by asking to be assigned to it, or by opening an empty PR. \n\nWhen opening an PR, please reference this issue using any of the [closing keywords](https://help.github.com/en/articles/closing-issues-using-keywords). \n\nIf you had a look at the exercise description and you concluded that the exercise might not be possible to implement in the Java language, please leave a comment and describe the problem. \n\nIn case you have any further questions, feel free to ask here. \n"
```

Comment:
I think I would change "If you had a look" to "If you have had a look", just to be more clear 🙂